app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create 
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+  def edit 
+    @user= User.find_by(id:params[:id])
+  end
+  def update
+    @user= User.find_by(id:params[:id])
```

Comment:
[OPTIONAL]  same as line 16: 
You can use a better query method called `find` ex.  `@user = User.find(params[:id])`